source/_components/binary_sensor.tod.markdown

```diff
@@ -0,0 +1,77 @@
+The `tod` platform supports binary sensors which get their values by checking
```

Comment:
Our whole documentation header is missing here, please be sure to check out developers documentation on how to add a new page (or simply look at some other example in this repository).

Reply:
Nevermind, fixed already.